js/app.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+
+   $('#submit-btn').click(function(event){ 
+      event.preventDefault();
+      var city = $('#city-type').val();
+   
+    
+	 if (city === 'New York' || city === 'New York City' || city === 'NYC') {
+    // make the page background image of New York City if one of these 3 terms are entered
+      $('body').removeClass(); $('body').addClass('nyc');
+    
+   } else if(city === 'San Fransisco' ||  city === 'SF' || city === 'Bay Area') {
+    // make the page background image of San Fransisco if one of these 3 terms are entered
+      $('body').removeClass(); $('body').addClass('sf');
+ 
+   } else if(city === 'LA' ||  city === 'LAX') {
+    // make the page background image of Los Angeles if one of these 3 terms are entered
+      $('body').removeClass(); $('body').addClass('la');
+
+   } else if(city === 'Austin' || city === 'ATX') {
+    // make the page background image of Austin if one of these 3 terms are entered
+      $('body').removeClass(); $('body').addClass('austin');
+
+   } else if(cityy === 'Sydney' || city === 'SYD') {
```

Comment:
Watch your spelling! One wrong letter and the whole thing goes kablooey! (Is that how you spell it??)